boardgame-ui/src/locales/translations.csv

```diff
@@ -209,3 +214,13 @@ event-board-games.toast.add-library-success,Board game successfully added to you
 event-board-games.toast.add-library-error,Failed to add the board game to your library,
 event-board-games.toast.delete-success,Board game successfully deleted,
 event-board-games.toast.delete-error,Failed to delete the board game,
+wish-list.title,My wish list,
+wish-list.subtitle,Keep track of the board games you want to play,
+wish-list.toast.add-success,Board game successfully added to wish list,
+wish-list.toast.add-error,Failed to add the board game to wish list,
+wish-list.toast.delete-success,Board game successfully removed from wish list,
+wish-list.toast.delete-error,Failed to remove the board game from wish list,
+matchmaking.no-result,Could not find any relevant matchmaking for you. Consider adding more games to your wish list.,
+matchmaking.info-message,"This page lists the games of your wish list available at this event, and to which other event attendees want to play",
+matchmaking.some-games-other-attendees-want-to-play,Board games other attendees want to play,
+matchmaking.other-attendees,Other attendees that want to play it,
```

Comment:
Should we shorten the sentences here?

Like: 
- "Also interested in playing:" 
- "Other on-demand board games"